### PR TITLE
Add `osu!(lazer) Updates: December 9, 2024` news post

### PR DESCRIPTION
**Pull request by 0x84f**
[preview](https://osu1.roanh.dev/home/news/2024-12-09-osulazer-updates-december-9-2024)

### CODE REVIEW

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+layout: post
+title: "osu!(lazer) Updates: December 9, 2024"
+date: 2024-12-09 15:00:00 +0000
+---
+
+Cure your post-world cup depression with some brand new osu!(lazer) updates!
+
+![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/banner.jpg)
+
+While the community has been indulging themselves to some top of the line osu! gameplay for the past few weeks, the developers stayed hard at work, making osu!(lazer) even better.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:11]:
```suggestion
While top players have been hard at work making world cup history for the past few weeks, the developers also stayed hard at work, making osu!(lazer) even better.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu!(lazer) Updates: December 9, 2024"
+date: 2024-12-09 15:00:00 +0000
+---
+
+Cure your post-world cup depression with some brand new osu!(lazer) updates!
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:7]:
```suggestion
Cure your post–world cup depression with some brand-new osu!(lazer) updates!
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+While the community has been indulging themselves to some top of the line osu! gameplay for the past few weeks, the developers stayed hard at work, making osu!(lazer) even better.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/E7cslGdXKLk" frameborder="0" allowfullscreen></iframe>
+</div>
+
+### Editor changes
+
+- **The visual appearance of highlighted osu!mania notes has been improved.** Cleans up some sub-par bounding boxes for objects, and makes selections easier to see.
+- **osu!mania notes won't disappear when reaching their hit time.** Another small annoyance that didn't make sense in the editor.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:20]:
```suggestion
- **osu!mania notes won't disappear when scrolled to anymore.** Another small annoyance that didn't make sense in the editor.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- **Objects will stay selected if you control-click on empty space.** If your aim is lacking, we have you covered (in the editor).
+
+### Gameplay changes
+
+- **The idle time before gameplay loads has been increased when hovering controls.** Now there's more time to ponder whether the background dim percentage is just right.
+- **A toggle has been added for star fountains during gameplay.** No more star fountain jump scares.
+
+![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/star-fountain.jpg)
+
+- **Your score will be saved even if you quick-restart after passing a map.** R.I.P. to all the sub-par runs that could've been saved so far, at least until they've been superseded.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:31]:
```suggestion
- **Your score will be saved even if you quick-restart after passing a map.** Rest in peace to all the subpar runs that were tossed into the void so far.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+- **The "watch replay" button now works on first press.** No, your mouse wasn't broken after all.
+- **A new osu! gameplay mod has been implemented, "Bloom".** They say everything's bigger in... osu!(lazer)?
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:33]:
```suggestion
- **New "Bloom" mod has been added.** They say everything's bigger in... osu!(lazer)?
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-12-09-mod-bloom.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Stage line alignment in osu!(lazer) now matches osu!(stable).** It's small things like these that can make all the difference for players.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:41]: idk if pixel-level adjustments are worth noting <.<

```suggestion
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- **You can now get an SS in osu!mania even if there are GREAT judgements.** It's big things like these that *do* make all the difference for players.
+- **The scroll bar for adjusting osu!mania scroll speed is now more granular.** The speed can now be adjusted with single decimal precision.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:43]:
```suggestion
- **The scroll bar for adjusting osu!mania scroll speed is now more granular.** The speed can now be adjusted with single-decimal precision.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+- **osu!taiko's constant scroll algorithm has been adjusted.** The speed more closely matches player expectations.
+
+### Other changes
+
+- **You can now install osu!(lazer) directly from osu!(stable)'s cutting edge release.** While not necessarily an osu!(lazer) change, it shows that slowly, but surely, osu!(lazer) is getting into a state ready for major transition.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:48]: mm long

```suggestion
- **You can now install osu!(lazer) directly from osu!(stable)'s cutting edge release.** While maybe still not the biggest nudge for people to transition, we're getting there.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/try-lazer.jpg)
+
+- **The public chats are now sorted alphabetically, and private chats are sorted based on recent messages.** Order has been... added?
+- **Your rank is now accurately presented based on the mode you are playing.** Previously only the rank of your default game mode would be visible in-game.
+- **Playlists can now be closed within a grace period after creation.** If you spot a mistake or change your mind on a playlist within the first 5 minutes, the playlist can be closed with the new `Close` button.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:54]:
```suggestion
- **Playlists can now be closed within 5 minutes after creation.** A more sane alternative than having to poke peppy.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+- **Hold-for-pause during gameplay has been re-added.** This time it's a toggle, meaning that sometimes you *can* get the best of both worlds.
+
+![](/wiki/shared/news/2024-12-09-osulazer-updates-december-9-2024/hold-to-pause.jpg)
+
+- **A map's maximum achievable pp can now be displayed with a skin element.** Select the `MaxPP` attribute for the `BeatmapAttributeText` skin element in the skin editor, and voilà.
+- **Player settings have been added to the spectator and replay screens.** This means that you can now adjust the same settings as if you were the one playing.
```
Comment (Walavouchey) [inline news/2024/2024-12-09-osulazer-updates-december-9-2024.md:60]: replays always had these settings

```suggestion
- **Gameplay settings have been added to the spectator screens.** This is the same menu that appears before playing a map or while watching a replay.
```

File: news/2024/2024-12-09-osulazer-updates-december-9-2024.md (hunk 6 above)
Comment (0x84f) [reply news/2024/2024-12-09-osulazer-updates-december-9-2024.md:41]: fair

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**